test/performance/performance-tests.sh

```diff
@@ -24,7 +24,7 @@ export BENCHMARK_ROOT_PATH="$GOPATH/src/knative.dev/serving/test/performance/ben
 source vendor/knative.dev/test-infra/scripts/performance-tests.sh
 
 function update_knative() {
-  local istio_version="istio-1.2-latest"
+  local istio_version="istio-1.4-latest"
```

Comment:
@chizhg I'm not 100% sure if performance test also updates to Istio 1.4 or not. If I should not update here, please let me know.